default/data/ui/views/rw_vc_zoom_ops.xml

```diff
@@ -1,8 +1,8 @@
 <dashboard theme="dark" stylesheet="style.css">
   <label>Zoom Ops</label>
   <search id="zoom_stats">
-    <query>`rw_vc_indexes` sourcetype="zoom:webhook" source="meeting.started" OR source="meeting.ended" 
-| eval x_{source}_time = _time 
+    <query>`rw_vc_indexes` sourcetype="zoom:webhook" event="meeting.started" OR event="meeting.ended" 
```

Comment:
Change `event="meeting.started"` to `event="meeting.started" OR event="webinar.started" OR event="meeting.ended" OR event="webinar.ended"` to account for webinars.